docs/submission/submitted-data.md

```diff
@@ -2,103 +2,126 @@
 id: submitted-data
 title: Viewing Submitted Data
 ---
-Program members can track the progress of their program’s clinical and molecular data submission and download analysis files from the **Program Dashboard**. 
 
-When a program member logs in to the ARGO Data Platform, and navigates to the **Program Services** area in the top menu, they will land on the program dashboard.  If a member belongs to multiple programs, each program has its own dashboard and submission sections.
+Program members can track the progress of their program’s clinical and molecular data submission and download analysis files from the **Program Dashboard**.
 
-Program progress is summarized within the following dashboard "cards": 
+When a program member logs in to the ARGO Data Platform, and navigates to the **Program Services** area in the top menu, they will land on the program dashboard. If a member belongs to multiple programs, each program has its own dashboard and submission sections.
 
-## Program Statistics 
+Program progress is summarized within the following dashboard "cards":
+
+## Program Statistics
 
 This card aggregates the following program statistics:
-![Dashbard Program Statistics](/assets/submission/dashboard-stats.png)  
+![Dashbard Program Statistics](/assets/submission/dashboard-stats.png)
 
 #### Registered Donors
+
 Once samples are registered, each corresponding donor will be counted as a percentage out of the number of committed program donors.
 
 #### % Donors with all Core Clinical Data
-Once all core clinical fields and files have been submitted for a donor, they are considered **"clinically complete"**. 
+
+Once all core clinical fields and files have been submitted for a donor, they are considered **"clinically complete"**.
+
 > **A donor must be clinically complete before any of their molecular analysis files are released to the program members.**
 
-Check out [Clinical Data Validation Rules](clinical-data-validation-rules#clinical-data-completeness) for more information on how clinical completeness is calculated.
+Check out [Clinical Data Validation Rules](/docs/submission/clinical-data-validation-rules#clinical-data-completeness) for more information on how clinical completeness is calculated.
 
 #### % Donors with Tumour & Normal
-*Coming Soon:* Once a donor has a tumour and normal raw read pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
+
+_Coming Soon:_ Once a donor has a tumour and normal raw read pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
 
 #### Donors in Molecular Data Processing
-*Coming Soon:* This is the number of donors that currently have samples being processed in the [analysis pipeline](dna-pipeline).
+
+_Coming Soon:_ This is the number of donors that currently have samples being processed in the [analysis pipeline](dna-pipeline).
```

Comment:
probably need to change a lot of link references with the new folder structure right? like this one

Reply:
yup looks like i missed these two in my review!

Reply:
yeah I can test the links once merged too...hard to test this way :)